RetakesPlugin/Modules/Configs/RetakesConfigData.cs

```diff
@@ -9,6 +9,7 @@ public class RetakesConfigData
     public float TerroristRatio { get; set; } = 0.45f;
     public int RoundsToScramble { get; set; } = 5;
     public bool IsScrambleEnabled { get; set; } = true;
+    public bool AutoTeamBalance { get; set; } = true;
```

Comment:
When altering this file, also make sure to bump the version at the top as I have an auto config migration script